TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
+1b)	"1" === 1;  false;  using === means that the comparison includes not only equal value but equal type.  (here is string versus number)
+1c)	 x == 'x';	false;  string 'x' is converted tonumber(x) which is different than the number x
+1d)	 x == (x+''); true;  number is equal to that number + an empty string
+1e)	'' == ' ';	false;  empty sting is always false but ' ' is a valid string
+1f)	x = true;	true;	assignment variable can assign x = true
+1g)	var x; x == 'undefined';	false;	undefined is equal to null but not a true value
+1h)	'9'<'10'; false;	These both evaluate to a string so they both are equal
```

Comment:
In this case, what is being evaluated here is whether or not they are valid strings, but rather the numerical values of their characters. The < comparator is checking '9' vs '10' one character at a time. In this situation '9' is greater than '1' so javascript stops comparison and returns false. Similarly, '950' < '9256' returns false as well, because '9' == '9', which moves to the next character set, returning '5' > '2'. As soon as javascript finds one character greater than another, it discontinues comparing the sets, and returns with a boolean. This works with alpha strings as well. For example, 'b' > 'a’!